runners/pom.xml

```diff
@@ -35,6 +35,7 @@
   <modules>
     <module>core-construction-java</module>
     <module>core-java</module>
+    <module>harness-java</module>
```

Comment:
Shouldn't this additional module only be executed when the java8 profile is enabled?

Reply:
yup. Done.